docs/docs/behaviors/misc.md

```diff
@@ -40,3 +40,22 @@ Example:
 ```
 &none
 ```
+
+## Sleep
+
+The sleep behavior simply puts your keyboard to sleep. This behavior has the same effect as letting the sleep timer expire.
+
+:::note
+You'll need to enable `CONFIG_ZMK_SLEEP` in your boards config for this keycode to work.
+:::
+
+### Behavior Binding
+
+- Reference: `&sleep`
+- Parameters: None
+
+Example:
+
+```
+&sleep
+```
```

Comment:
Please document this as being a global behavior, that will affect all components of a split. See other global behaviors, e.g. RGB, for an example docs.